src/twisted/python/_setup.py

```diff
@@ -121,23 +122,41 @@ def __init__(self, *args: object, **kwargs: object) -> None:
         Extension.__init__(self, *args, **kwargs)
 
 
+def _checkCPython(platform: Any = platform) -> bool:
+    """
+    Checks if this implementation is CPython.
+
+    This uses C{platform.python_implementation}.
+
+    This takes C{sys} and C{platform} kwargs that by default use the real
+    modules. You shouldn't care about these -- they are for testing purposes
+    only.
+
+    @return: C{False} if the implementation is definitely not CPython, C{True}
+        otherwise.
+    """
+    return cast(bool, platform.python_implementation() == "CPython")
+
+
+_isCPython = _checkCPython()  # type: bool
+
+
 # The C extensions used for Twisted.
-_EXTENSIONS = [
-    ConditionalExtension(
-        "twisted.test.raiser",
-        sources=["src/twisted/test/raiser.c"],
-        condition=lambda _: _isCPython,
-    ),
-    ConditionalExtension(
-        "twisted.internet.iocpreactor.iocpsupport",
-        sources=[
-            "src/twisted/internet/iocpreactor/iocpsupport/iocpsupport.c",
-            "src/twisted/internet/iocpreactor/iocpsupport/winsock_pointers.c",
-        ],
-        libraries=["ws2_32"],
-        condition=lambda _: _isCPython and sys.platform == "win32",
-    ),
-]
+_EXTENSIONS = (
+    [
+        ConditionalExtension(
+            "twisted.internet.iocpreactor.iocpsupport",
+            sources=[
+                "src/twisted/internet/iocpreactor/iocpsupport/iocpsupport.c",
+                "src/twisted/internet/iocpreactor/iocpsupport/winsock_pointers.c",
+            ],
+            libraries=["ws2_32"],
+            condition=lambda _: True,
```

Comment:
```suggestion
            condition=lambda _: _isCPython and sys.platform == "win32",
```

Reply:
If condition is alwasy True, this can be replaced with a normal extension 
We can remove the usage of ConditionalExtension

Reply:
there's a load of stuff in there I don't understand about solaris, so I hesitated to remove it
https://github.com/twisted/twisted/blob/4a74c436e4d359f88ca5ff602c22ed740fe5012b/src/twisted/python/_setup.py#L235-L244

I think it can all go?  what's the WIN32 macro for?

Reply:
solaris can be removed as Twisted no longer supports Solaris